### DISCUSSION TITLE
Spotify Integration

### DISCUSSION THREAD

**Discussion by andrewmcgivery** [andrewmcgivery/obsidian-soundscapes#16] [Ideas]
Trying to gage interest in a Spotify integration.

As I am doing research, it would seem that playback outside of official Spotify apps requires a Premium subscription.

Are you interested in this integration? How would you like to use it? Do you have a Premium Spotify subscription?

**Comment by ThomasEricB** [comment]
I have a premium spotify! I would love such a feature

**Comment by kennygokh** [comment]
Yes. I would love to.
